stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -365,6 +365,7 @@ internal class ChannelControllerImpl(
             return
         }
         runChannelQuery(QueryChannelPaginationRequest(limit))
+        errorWatching = recoveryNeeded
```

Comment:
what's the difference between these 2 variables?

Reply:
Watch is only the first time. When you load more messages (older or newer), if they fails, we don't need to watch it again